tests/snmpsim/eltek.snmprec

```diff
@@ -0,0 +1 @@
+.1.3.6.1.4.1.12148.10.2.4.0|4|Eltek
```

Comment:
Is this the one that doesn't have a sysObjectId or sysDescr?

Reply:
Yes indeed.